src/models/utils/selectablefilter.ts

```diff
@@ -0,0 +1,6 @@
+import { Filter } from '@yext/answers-core';
+
+export interface SelectableFilter {
+  filter: Filter,
+  selected: boolean
```

Comment:
I know Connor already commented something similar, but I feel a little hesitant calling this a "SelectableFilter" if it doesn't extend "Filter". I think I would prefer just having it extend "Filter", since otherwise the state is a little more nested than it might need to be

Reply:
Also thanks to destructuring I don't think it should be tooo ugly to format the filters for core?

Reply:
hmm ok, I'm fine either way. yeah it would be more consistent with the facets interface like connor said. Since the team prefer the extend approach, I can make the change!

Reply:
updated